### PR TITLE
[Wearable] Replace "change" event with "input" for Slider

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] N/A

[Problem] Mobile profile successfully emits both events
	"input" and "change" on Slider widget whereas Wearable profile
	emits only "change".

[Solution] Emits "input" event instead of "change" since according to [1]
	input event should be fired immediately when the slider is adjusted but
	change event does not fire immediately, the value does not change until
	the slider stops moving
	This fixes SThing.js behaviour with Slider and Dimmer synchronization
	for Wearable.

[Remarks] May cause regression for tau apps already using "change" event.
	To fix this we should provide real implementation for "change" event.

[1] https://www.impressivewebs.com/onchange-vs-oninput-for-range-sliders/

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Review by lslachciak — CHANGES_REQUESTED** [review]
@grzegorz-czajkowski please make sure that CHANGE is not used by UIComponents.
After check, I'll approve

**Comment by pkaczmarczy** [comment]
The change event is used in tau 1.1 wearable dimmer sample.
I think we should trigger both events.

IMO change event should be triggered after user release the slider handle and value changes, and input event should be triggered with each handle movement.

**Comment by grzegorz-czajkowski** [comment]
@lslachciak 
"change" event is being used by
examples/wearable/UIComponents/contents/controls/slider/extended_slider.js
in both tau 1.0 and 1.1.
I will fix this.

@pkaczmarczy 
Where I can find source of  tau 1.1 wearable dimmer sample?
>>  I think we should trigger both events.
They behave slightly different, if someone needs "change" event we should implemented it according to spec.

**Review by lslachciak — APPROVED** [review]

**Comment by lslachciak** [comment]
> 
> 
> The change event is used in tau 1.1 wearable dimmer sample.
> I think we should trigger both events.
> 
> IMO change event should be triggered after user release the slider handle and value changes, and input event should be triggered with each handle movement.


@pkaczmarczy 
I think we can create new issue for change event implementation in future version of TAU not blocking this one. OK?

**Comment by pkaczmarczy** [comment]
Yes, I think we should fix event emitting for slider.
@grzegorz-czajkowski The dimmer sample is available only on tau_1.1 branch and file path is: examples/wearable/UIComponents/contents/controls/dimmer.js

**Comment by grzegorz-czajkowski** [comment]
@pkaczmarczy Ok thanks. I will prepare separate PR to fix this.

**Comment by grzegorz-czajkowski** [comment]
@lslachciak  and @pkaczmarczy 
Created issue to add "change" event - https://github.com/Samsung/TAU/issues/453
Patch to tau 1.1 is here https://github.com/Samsung/TAU/pull/449

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]
In my opinion issue #453 is blocking for this PR

**Comment by grzegorz-czajkowski** [comment]
This needs improvements and is out of scope currently.

**Pull request closed without merging**
